inc/class_parser.php

```diff
@@ -1574,7 +1574,7 @@ function mycode_parse_video($video, $url)
 					$id = $path[1]; // http://xy.screen.yahoo.com/fds-123.html
 				}
 				// Support for localized portals
-				if(!empty($parsed_url['host']))
+                if(is_array($parsed_url) && array_key_exists($parsed_url['host']))
```

Comment:
This also seems to be already addressed.

You can also replace whitespaces with tabs in this PR for consistency.